hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/OzoneManagerServiceProviderImpl.java

```diff
@@ -167,6 +167,30 @@ public OzoneManagerServiceProviderImpl(
     this.reconTaskStatusDao = reconTaskController.getReconTaskStatusDao();
     this.ozoneManagerClient = ozoneManagerClient;
     this.configuration = configuration;
+
+    registerOMDBTasks();
```

Comment:
Can we avoid calling this from a constructor? Is there a lifecycle method which can do the same thing? @avijayanhwx

Reply:
There is a start() method where this can be done.